src/components/Articles/ResolutionDates.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div id="resolution-dates">
+    <template>
+
+      <!-- Base Display -->
+      <v-row no-gutters class="mb-4">
+        <v-col cols="3">
+          <label>
+            <span>Resolution or <br> Court Order Dates</span>
+          </label>
+        </v-col>
+
+        <v-col cols="7">
+          <span class="info-text">Indicate the date of the resolution or court order used to alter the company's share
+            structure or the special rights and restrictions of a class or series of shares:</span>
+        </v-col>
+
+        <v-col cols="2" class="align-right">
+          <v-btn v-if="!isEditing"
+                 id="add-resolution-date"
+                 class="add-btn"
+                 text color="primary"
+                 @click="isEditing = true"
+          >
+            <v-icon small>mdi-plus</v-icon>
+            <span>Add</span>
+          </v-btn>
+        </v-col>
+      </v-row>
+
+      <!-- Display NEW Dates -->
+      <v-row no-gutters v-if="addedDates && addedDates.length >= 1" class="my-4">
+        <v-col cols="3"></v-col>
+        <v-col cols="7">
+          <ul class="resolution-date-list info-text pl-0 mt-2">
+            <li v-for="(date, index) in addedDates"
+                :key="`newResolutionDate: ${index}`"
```

Comment:
This likely works but it may look better to have keys such as `newResolutionDate-1`. Thoughts?

Reply:
Also line 81?

Reply:
Yes, i think you are right. Will update!